eng/versioning/version_client.txt

```diff
@@ -124,7 +124,7 @@ com.azure:azure-data-schemaregistry-jsonschema;1.0.0;1.1.0-beta.1
 com.azure:azure-data-tables;12.5.0;12.6.0-beta.1
 com.azure:azure-data-tables-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-developer-devcenter;1.0.5;1.1.0-beta.1
-com.azure:azure-developer-loadtesting;1.0.18;1.1.0-beta.1
+com.azure:azure-developer-loadtesting;1.0.20;1.0.20
```

Comment:
For releasing you'll want to only update the second version. Also, did 1.0.19 get skipped? If not, this should use version `1.0.19`.

```suggestion
com.azure:azure-developer-loadtesting;1.0.18;1.0.19
```